recipes/yaqd-core/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "yaqd-core" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2d40037fb0506294cc82f879ffe0baeb7ed1c3c96572f6587e2e920ecd261fc3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - yaqd-base=yaqd_core._daemon:Base.main
+    - yaqd-hardware=yaqd_core._hardware._base:Hardware.main
+    - yaqd-continuous-hardware=yaqd_core._hardware._continuous:ContinuousHardware.main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.7
+    - appdirs
+    - toml
+
+test:
```

Comment:
I think we should probably be testing the console scripts as well if feasible:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
The entry points start daemons that expect configuration files, so it's not super easy to test them...

Reply:
I could try to run our pytest directory---going to play with that now

Reply:
I don't want to hold up the recipe. If it's simple to just run something like `yaqd-base -h` for each of the console scripts without setting up a bunch of config files, that would be fine. We're mostly just testing for existence. If it's going to be a lot of work, let's just skip it. No need to run the full test suite.

Reply:
OK @synapticarbors I believe that's done :smile: thanks for your kind direction